python/custreamz/custreamz/tests/test_kafka.py

```diff
@@ -5,11 +5,10 @@
 from cudf.testing._utils import assert_eq
 
 
-@pytest.mark.parametrize("commit_offset", [-1, 0, 1, 1000])
+@pytest.mark.parametrize("commit_offset", [1, 45, 100, 22, 1000, 10])
```

Comment:
Why was this change relevant to this PR?

Reply:
Wanted to add extra test cases around possible values that could cause issues.